Warnings raised by `pydocstyle` and `pylint`

Pydocstyle
```
toqito/state_props/von_neumann_entropy.py:8 in public function `von_neumann_entropy`:
        D205: 1 blank line required between summary line and description (found 0)
toqito/state_props/von_neumann_entropy.py:8 in public function `von_neumann_entropy`:
        D400: First line should end with a period (not ' ')
toqito/matrix_props/is_totally_positive.py:7 in public function `is_totally_positive`:
        D400: First line should end with a period (not '`')
toqito/matrix_props/is_totally_positive.py:7 in public function `is_totally_positive`:
        D401: First line should be in imperative mood (perhaps 'Determine', not 'Determines')
toqito/states/mutually_unbiased_basis.py:1 at module level:
        D100: Missing docstring in public module
toqito/states/mutually_unbiased_basis.py:8 in public function `mutually_unbiased_basis`:
        D202: No blank lines allowed after function docstring (found 1)
toqito/states/mutually_unbiased_basis.py:8 in public function `mutually_unbiased_basis`:
        D401: First line should be in imperative mood; try rephrasing (found 'Function')
toqito/channel_props/is_unital.py:[17](https://github.com/vprusso/toqito/actions/runs/6989993559/job/19019055191?pr=228#step:7:18) in public function `is_unital`:
        D205: 1 blank line required between summary line and description (found 0)
toqito/channel_props/is_unital.py:17 in public function `is_unital`:
        D400: First line should end with a period (not 'm')
toqito/channel_props/is_trace_preserving.py:[18](https://github.com/vprusso/toqito/actions/runs/6989993559/job/19019055191?pr=228#step:7:19) in public function `is_trace_preserving`:
        D[20](https://github.com/vprusso/toqito/actions/runs/6989993559/job/19019055191?pr=228#step:7:21)5: 1 blank line required between summary line and description (found 0)
toqito/channel_props/is_trace_preserving.py:18 in public function `is_trace_preserving`:
        D400: First line should end with a period (not 'm')
```

pylint

```
toqito/state_opt/tests/test_state_exclusion.py:2:0: W0611: Unused numpy imported as np (unused-import)
************* Module state_props.tests.test_is_antidistinguishable
toqito/state_props/tests/test_is_antidistinguishable.py:2:0: W0611: Unused numpy imported as np (unused-import)
************* Module perms.permute_systems
toqito/perms/permute_systems.py:2[14](https://github.com/vprusso/toqito/actions/runs/6989993559/job/19019055191?pr=228#step:6:15):0: I0021: Useless suppression of 'redefined-variable-type' (useless-suppression)
```

--- a/toqito/channel_props/is_trace_preserving.py
+++ b/toqito/channel_props/is_trace_preserving.py
@@ -16,9 +16,8 @@ def is_trace_preserving(
     dim: list[int] | np.ndarray = None,
 ) -> bool:
     r"""
-    Determine whether the given channel is trace-preserving (Section: Linear Maps Of Square Operators from
-    :cite:`Watrous_2018_TQI`).
-
+    Determine whether the given channel is trace-preserving.
+     
     A map :math:`\Phi \in \text{T} \left(\mathcal{X}, \mathcal{Y} \right)` is
     *trace-preserving* if it holds that
 
@@ -76,12 +75,14 @@ def is_trace_preserving(
     >>> is_trace_preserving(choi_mat)
     True
 
+    Further information for determining the trace preserving properties of channels consult (Section: Linear Maps Of
+    Square Operators from :cite:`Watrous_2018_TQI`).
+
     References
     ==========
     .. bibliography::
         :filter: docname in docnames
     
-
     :param phi: The channel provided as either a Choi matrix or a list of Kraus operators.
     :param rtol: The relative tolerance parameter (default 1e-05).
     :param atol: The absolute tolerance parameter (default 1e-08).

--- a/toqito/channel_props/is_unital.py
+++ b/toqito/channel_props/is_unital.py
@@ -1,6 +1,4 @@
 """Determine whether channel is unital."""
-
-
 import numpy as np
 
 from toqito.channel_ops import apply_channel
@@ -15,11 +13,9 @@ def is_unital(
     dim: int | list[int] | np.ndarray = None,
 ) -> bool:
     r"""
-    Determine whether the given channel is unital (Chapter: Unital Channels And Majorization from
-    :cite:`Watrous_2018_TQI`).
+    Determine whether the given channel is unital.
 
-    A map :math:`\Phi \in \text{T} \left(\mathcal{X}, \mathcal{Y} \right)` is *unital* if it
-    holds that
+    A map :math:`\Phi \in \text{T} \left(\mathcal{X}, \mathcal{Y} \right)` is *unital* if it holds that:
 
     .. math::
         \Phi(\mathbb{I}_{\mathcal{X}}) = \mathbb{I}_{\mathcal{Y}}.
@@ -27,6 +23,8 @@ def is_unital(
     If the input channel maps :math:`M_{r,c}` to :math:`M_{x,y}` then :code:`dim` should be the
     list :code:`[[r,x], [c,y]]`. If it maps :math:`M_m` to :math:`M_n`, then :code:`dim` can simply
     be the vector :code:`[m,n]`.
+
+    More information can be found in Chapter: Unital Channels And Majorization from :cite:`Watrous_2018_TQI`).
 
     Examples
     ==========

--- a/toqito/matrix_props/is_totally_positive.py
+++ b/toqito/matrix_props/is_totally_positive.py
@@ -5,7 +5,7 @@ import numpy as np
 
 def is_totally_positive(mat: np.ndarray, tol: float = 1e-6, sub_sizes: list | None = None):
     r"""
-    Determines whether a matrix is totally positive. :cite:`WikiTotPosMat`
+    Determine whether a matrix is totally positive. :cite:`WikiTotPosMat`.
 
     A totally positive matrix is a square matrix where all the minors are positive. Equivalently, the determinant of
     every square submatrix is a positive number.

--- a/toqito/state_opt/tests/test_state_exclusion.py
+++ b/toqito/state_opt/tests/test_state_exclusion.py
@@ -1,5 +1,4 @@
 """Test state_exclusion."""
-import numpy as np
 import pytest
 
 from toqito.states import bell

--- a/toqito/state_props/tests/test_is_antidistinguishable.py
+++ b/toqito/state_props/tests/test_is_antidistinguishable.py
@@ -1,5 +1,4 @@
 """Test is_antidistinguishable."""
-import numpy as np
 import pytest
 
 from toqito.states import bell, trine

--- a/toqito/state_props/von_neumann_entropy.py
+++ b/toqito/state_props/von_neumann_entropy.py
@@ -6,8 +6,7 @@ from toqito.matrix_props import is_density
 
 def von_neumann_entropy(rho: np.ndarray) -> float:
     r"""
-    Compute the von Neumann entropy of a density matrix :cite:`WikiUVonNeumann`, 
-    Section: "Definitions Of Quantum Entropic Functions" from :cite:`Watrous_2018_TQI`).
+    Compute the von Neumann entropy of a density matrix :cite:`WikiUVonNeumann`.
 
     Let :math:`P \in \text{Pos}(\mathcal{X})` be a positive semidefinite operator, for a complex
     Euclidean space :math:`\mathcal{X}`. Then one defines the *von Neumann entropy* as
@@ -24,6 +23,9 @@ def von_neumann_entropy(rho: np.ndarray) -> float:
     where the :math:`\text{log}` function is assumed to be the base-2 logarithm, and where
     :math:`\Sigma` is an alphabet where :math:`u \in [0, \infty)^{\Sigma}` is a vector of
     nonnegative real numbers indexed by :math:`\Sigma`.
+
+    Further information for computing the von Neumann entropy of a density matrix can be found in Section: "Definitions
+    Of Quantum Entropic Functions" from :cite:`Watrous_2018_TQI`).
 
     Examples
     ==========

--- a/toqito/states/mutually_unbiased_basis.py
+++ b/toqito/states/mutually_unbiased_basis.py
@@ -1,3 +1,4 @@
+"""Construct a set of mutually unbiased bases."""
 import numpy as np
 from sympy import primerange, isprime
 
@@ -6,7 +7,7 @@ from toqito.matrices import gen_pauli
 
 def mutually_unbiased_basis(dim: int) -> list[np.ndarray]:
     r"""
-    Function to generate list of MUBs for a given dimension :cite:`WikiMUB`.
+    Generate list of MUBs for a given dimension :cite:`WikiMUB`.
     
     Note that this function only works if the dimension provided is prime or a power of a prime. Otherwise, we don't
     know how to generate general MUBs.
@@ -29,7 +30,6 @@ def mutually_unbiased_basis(dim: int) -> list[np.ndarray]:
     :param dim: The dimension of the mutually unbiased bases to produce.
     :return: The set of mutually unbiased bases of dimension :code:`dim` (if known).
     """
-    
     # The first basis will always be the standard basis:
     mats = [np.eye(dim)]
 

